content/en/docs/zero-code/python/_index.md

```diff
@@ -23,6 +23,12 @@ opentelemetry-bootstrap -a install
 The `opentelemetry-distro` package installs the API, SDK, and the
 `opentelemetry-bootstrap` and `opentelemetry-instrument` tools.
 
+> **NOTE:** You need to install a distro package to get auto instrumentation
+> working. The `opentelemetry-distro` package contains the default distro and
+> configurator to automatically configures some of the common options for users.
+> For more information about `opentelemetry-distro` check
+> [here](/docs/languages/python/distro/).
```

Comment:
Could you also convert this Markdown note to short code? Here's an [example](https://github.com/open-telemetry/opentelemetry.io/blob/ada8fd7a9d47e188f5e7c94e7d9affd7c0588c5e/content/en/docs/zero-code/js/_index.md?plain=1#L48). And while we're at it, we should change the second note on the page as well.

```suggestion
> **NOTE:** You need to install a distro package to get auto instrumentation
> working. The `opentelemetry-distro` package contains the default distro and
> configurator to automatically configures some of the common options for users.
> For more information about `opentelemetry-distro` check
> [here](/docs/languages/python/distro/).
```

Reply:
Thanks for the review. Will update the notes then